src/components/HFUI/HFUI.js

```diff
@@ -136,6 +137,7 @@ const HFUI = ({
           )}
           <BadConnectionModal />
           <CcyInfoModal />
+          <ConfirmDMSModal />
```

Comment:
change this to only show modal in electron-mode, for near future we don't plan to allow DMS updates from hosted SPA